js/ui/filter_builder/utils.js

```diff
@@ -524,6 +620,47 @@ function setFocusToBody() {
     }
 }
 
+function getBetweenConfig(editorTemplate) {
```

Comment:
The between operation implementation (config and editor) may be moved to separate file.